app/scripts/components/exploration/tour-manager.tsx

```diff
@@ -200,3 +213,37 @@ export function PopoverTourComponent(props: ExtendedPopoverContentProps) {
     </Popover>
   );
 }
+
+
+// Tour invoking button is not strictly a map control (It does nothing re: map) 
+// but it needs to be styled as one of map control
+// This is why these componets stay in this file, not where all other map control lives
+export function TourButtonComponent({onClick}: {
+  onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+}) {
+  const datasets = useAtomValue(timelineDatasetsAtom);
+  const datasetCount = datasets.length;
+  return (
+  <SelectorButton
+    tipContent='Open guided tour'
+    tipProps={{ placement: 'left' }}
+    disabled={datasetCount === 0}
+    onClick={onClick}
+  >
+    <CollecticonCircleQuestion />
+  </SelectorButton>);
+}
+
+export function TourManagerInvokingButton(props) {
+  const { setIsOpen, setCurrentStep, setSteps } = useTour();
+  const reopenTour = useCallback(() => {
+    setCurrentStep(0);
+    setSteps?.(introTourSteps);
+    setIsOpen(true);
+  },[setIsOpen]);
+
+  useThemedControl(() => <TourButtonComponent {...props} onClick={reopenTour} />, {
+    position: 'top-right'
+  });
+  return null;
+}
```

Comment:
🔧 : can we get rid of passing in the rest of the props? seems like we just need to pass in `onClick`. Can we also move the jotai hooks from `TourButtonComponent` into `TourManagerInvokingButton` so then `TourButtonComponent` would just take in another prop for the `count`? that way we can keep `TourButtonComponent` pure. This can start helping us move hooks and data management pieces into smart components and then allowing dumb/pure components to note care about those pieces. Then we dont do this right now but if we wanted to unit test components in the future, this approach would allow that much easier.

Reply:
(Sorry, I decided to move the file to where other controls are; it is map/tour-control.tsx` now.)
Yeah, this is a really thoughtful comment aligning with what we are trying to do with VEDA. 🙇